test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CustomSearchParamTests.cs

```diff
@@ -486,7 +485,29 @@ private async Task DeleteSearchParameterAndVerify(SearchParameter searchParam)
         {
             if (searchParam != null)
             {
-                await Client.DeleteAsync(searchParam);
+                // Clean up new SearchParameter
+                // When there are multiple instances of the fhir-server running, it could take some time
+                // for the search parameter/reindex updates to propogate to all instances. Hence we are
+                // adding some retries below to account for that delay.
+                int retryCount = 0;
+                bool success = true;
+                do
+                {
+                    success = true;
+                    retryCount++;
+                    try
+                    {
+                        await Client.DeleteAsync(searchParam);
+                    }
+                    catch (Exception exp)
+                    {
+                        _output.WriteLine($"Failed to delete searchParameter: {exp}");
```

Comment:
>exp [](http://example.com/codeflow?start=79&length=3)

Would it contain search parameter url or any other identifier?

Reply:
yes it does